### PR TITLE
🍰 2019/kw27/data_import_organizations

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

import organizations (except addresses and undocumented fields)

### Issues

- relates #692 

### Todo

- [X] None

### CODE REVIEW

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql [hunk 1]
```diff
@@ -103,35 +103,68 @@
...
+o.language              = organization.language,
+o.avatar                = replace(organization.logo, 'https://api-alpha.human-connection.org', ''),
+o.coverImg              = replace(organization.coverImg, 'https://api-alpha.human-connection.org', ''),
+o.deleted               = COALESCE(organization.deleted,false),
+o.disabled              = NOT organization.isEnabled,
+o.createdAt             = datetime(organization.createdAt.`$date`),
+o.updatedAt             = datetime(organization.updatedAt.`$date`)
+WITH o, organization
+MATCH (u:User {id: organization.userId})
+MERGE (u)-[:CREATED_ORGA]->(o)
```
Comment (roschaefer) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql]:
```suggestion
MERGE (u)-[:CREATED]->(o)
```

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql [hunk 2]
```diff
@@ -103,35 +103,68 @@
...
+MERGE (u)-[:OWNING_ORGA]->(o)
```
Comment (roschaefer) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql]:
```suggestion
MERGE (u)-[:OWNS]->(o)
```
I don't like abbreviations. At first you think abbreviations are cool, but they always create confusion.

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql [hunk 3]
```diff
@@ -103,35 +103,68 @@
...
+WITH o, organization
+MATCH (u:User {id: organization.reviewedBy})
+MERGE (u)-[:REVIEWED_ORGA]->(o)
```
Comment (roschaefer) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql]:
```suggestion
MERGE (u)-[:REVIEWED]->(o)
```

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql (hunk 1 above)
Comment (ulfgebhardt) [reply deployment/legacy-migration/maintenance-worker/migration/neo4j/organizations/organizations.cql]: The Reason this was implemented this way is, cuz the schema definition for those relations were already present in that particular way. I wanted to import with as less as possible changes.

**Comment by Tirokk** [comment]
I left a relation to this PR `🍰 2019/kw27/data_import_organizations #950` and the related branch at `🌟 [EPIC] Organizations#281` for later use @ulfgebhardt and close this PR for now. We decided to import the organisation data later.

**Pull request closed without merging**